cli/core/hack/rpm/tanzu-cli.spec

```diff
@@ -0,0 +1,46 @@
+Name:       tanzu-cli
+Version:    %{cli_version}
+Release:    0
+License:    Apache 2.0
+URL:        https://github.com/vmware-tanzu/tanzu-cli/
+Vendor:     VMware
+Summary:    The Tanzu CLI
+
+%ifarch amd64
+%define arch amd64
+%endif
+
+%ifarch aarch64
+# TODO For now, we use the amd64 build for arm64
+%define arch amd64
+%endif
+
+%undefine _disable_source_fetch
+Source0:    https://github.com/vmware-tanzu/tanzu-framework/releases/download/v%{version}/tanzu-cli-linux-%{arch}.tar.gz
+
+%description
+VMware Tanzu is a modular, cloud native application platform that enables vital DevSecOps outcomes
+in a multi-cloud world.  The Tanzu CLI allows you to control VMware Tanzu from the command-line.
+
+# Go does not generate a build-id compatible with RPM, so we disable the need for a build-id
+# See https://github.com/rpm-software-management/rpm/issues/367
+%global _missing_build_ids_terminate_build 0
+
+# This is required to avoid some missing debug file errors
+%define debug_package %nil
+
+%prep
+%setup -T -c
+
+%build
+# Nothing to build
+
+%install
+rm -rf $RPM_BUILD_ROOT
+mkdir -p $RPM_BUILD_ROOT/%{_bindir}
+# Unpack the CLI and rename it
+tar -xzf %{SOURCE0} --strip-components 1 v%{version}/tanzu-core-linux_%{arch}
```

Comment:
we can do the extraction in %prep step ?

Reply:
Maybe, but when I started, I had less control of where the extraction would happen.
If you are familiar with the %prep syntax, I'm open to improving

Reply:
can the replace the section as:

```
%prep
%setup -q -n v%{version}

%build
# Nothing to build

%install
rm -rf $RPM_BUILD_ROOT
mkdir -p $RPM_BUILD_ROOT/%{_bindir}
mv tanzu-core-linux_%{arch} $RPM_BUILD_ROOT/%{_bindir}/tanzu
```

test:
1. build:
```bash
$ rpmbuild -v --define "cli_version 0.26.0" -bb tanzu-cli.spec --target amd64

...

Building for target amd64
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.1eQlxg
+ umask 022
+ cd /root/rpmbuild/BUILD
+ cd /root/rpmbuild/BUILD
+ rm -rf v0.26.0
+ /usr/bin/gzip -dc /root/rpmbuild/SOURCES/tanzu-cli-linux-amd64.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd v0.26.0
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
+ RPM_EC=0
++ jobs -p
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.zsYwnN

...

+ exit 0

RPM build warnings:
    Missing build-id in /root/rpmbuild/BUILDROOT/tanzu-cli-0.26.0-1.x86_64/usr/bin/tanzu
```
2. Install
```
$ rpm -ivh RPMS/amd64/tanzu-cli-0.26.0-1.amd64.rpm
Verifying...                          ################################# [100%]
Preparing...                          ################################# [100%]
Updating / installing...
   1:tanzu-cli-0.26.0-1               ################################# [100%]
```

3. Verify
```
[root@78a8a31a6b05 rpmbuild]# rpm -ql tanzu-cli
/usr/bin/tanzu
[root@78a8a31a6b05 rpmbuild]# tanzu version
version: v0.26.0
buildDate: 2022-09-27
sha: 10210a01
```

Reply:
Awesome!  Thanks.  I've updated and forced pushed a new version.